src/tools/singlejar/port.h

```diff
@@ -33,6 +35,8 @@ inline tm* localtime_r(const time_t* tin, tm* tout) {
   return nullptr;
 }
 
+// Make sure that the file HANDLE associated with |fd| is created by CreateFileA
```

Comment:
nit: `CreateFileW` is also fine, so how about just `CreateFile` instead of `CreateFileA`?

Reply:
Done